cmake/Install.cmake

```diff
@@ -605,6 +605,12 @@ if(BUILD_DOCS)
   install(FILES "${PROJECT_BINARY_DIR}/doc/pdf/PlantApplicationGuide.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
   install(FILES "${PROJECT_BINARY_DIR}/doc/pdf/UsingEnergyPlusForCompliance.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
   install(FILES "${PROJECT_BINARY_DIR}/doc/pdf/index.html" DESTINATION "./Documentation" COMPONENT Documentation)
+  if(WIN32)
```

Comment:
OK, so on Windows, I added some new [.url](https://fileinfo.com/extension/url) files that point to the docs on RTD.  These will show up in the install/Documentation folder right next to the PDFs so it shouldn't really affect the user much.  It will just open the default browser instead of the default PDF viewer.